### ISSUE TITLE
TypeError: Cannot read property 'forEach' of undefined

### ISSUE THREAD

**Issue by quasiperfect** [peterbe/minimalcss#231]
hi

what could trigger that error ?

i got this issue testing on http://codingheads.com/

**Comment by stereobooster** [comment]
Caused by:

```
@-moz-document url-prefix() {
	inputtype:radio],input[type=checkbox]{vertical-align:middle;
	margin-top:-.3em
}
}
```

Which is parsed to

```
{ type: 'Raw',
  loc: null,
  value: 'inputtype:radio],input[type=checkbox]' }
```

**Comment by quasiperfect** [comment]
@stereobooster thanks for your help. that would be great(handle invalid CSS)

**Comment by harrygreen** [comment]
This error actually found a ton of invalid crap in our production CSS!

**Comment by stereobooster** [comment]
fixed in #237
